### PR TITLE
`e2e` - updates tests

### PR DESCRIPTION
**Pull request by thedoublejay**
#### What kind of PR is this?:

/kind chore

#### What this PR does / why we need it:
2. Smoke - Change Passcode
3. Smoke - Check if API returns same value in DEX PoolPairs
4. Added tests for API errors
5. Updated Add Liquidity Tests
6. Updated Remove Liquidity Tests
#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: cypress/integration/functional/wallet/balances/balances.spec.ts [hunk 1]
```diff
@@ -26,3 +28,21 @@ context('Wallet - Balances', () => {
     cy.getByTestID('address_text').should('exist')
   })
 })
+
+context.only('Wallet - Balances - Failed API', () => {
```
Comment (ivan-zynesis) [inline cypress/integration/functional/wallet/balances/balances.spec.ts:32]: is the `.only` here on purpose?

File: cypress/integration/functional/wallet/dex/poolswap.spec.ts [hunk 2]
```diff
@@ -87,3 +87,21 @@ context('Wallet - DEX - Pool Swap with balance', () => {
     })
   })
 })
+
+context.only('Wallet - DEX - Pool Swap failed api', () => {
```
Comment (ivan-zynesis) [inline cypress/integration/functional/wallet/dex/poolswap.spec.ts:91]: another .only

File: cypress/integration/functional/wallet/balances/balances.spec.ts (hunk 1 above)
Comment (thedoublejay) [reply cypress/integration/functional/wallet/balances/balances.spec.ts:32]: ha

Comment (thedoublejay) [reply cypress/integration/functional/wallet/balances/balances.spec.ts:32]: my bad haha I'm testing this case only

File: .gitignore [hunk 3]
```diff
@@ -48,3 +48,5 @@ web-build/
 # Cypress
 /cypress/screenshots
 /cypress/videos
+
+/cypress-visual-screenshots/**
```
Comment (fuxingloh) [inline .gitignore:52]: the jpg to check against need to be committed

Comment (thedoublejay) [reply .gitignore:52]: @fuxingloh the QR code varies per run. It creates the snapshot first -> exit wallet -> load and check the snapshot

Comment (fuxingloh) [reply .gitignore:52]: Oh, I think we should have a fixed entered mnemonic and check against the QR code to detect regression in QR generation.

File: cypress/integration/functional/wallet/balances/receive.spec.ts [hunk 4]
```diff
@@ -25,3 +25,16 @@ context('Wallet - Receive', () => {
...
+context('Wallet - Receive - QR Code - Check', () => {
+  before(function () {
+    cy.createEmptyWallet()
+    cy.getByTestID('balances_list').should('exist')
+    cy.getByTestID('balances_row_0_utxo').click()
+    cy.getByTestID('receive_button').click()
+  })
+
+  it('should match QR code', function () {
+    cy.getByTestID('qr_code_container').compareSnapshot('qr-code-container')
```
Comment (fuxingloh) [inline cypress/integration/functional/wallet/balances/receive.spec.ts:38]: should check the address also, making sure it's the same address

Comment (thedoublejay) [reply cypress/integration/functional/wallet/balances/receive.spec.ts:38]: this one is handled on another spec

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**